packages/web/src/components/edit-track/EditTrackForm.tsx

```diff
@@ -141,7 +142,7 @@ const TrackEditForm = (
                 forceOpen={forceOpenAccessAndSale}
                 setForceOpen={setForceOpenAccessAndSale}
               />
-              <AdvancedField />
+              <AdvancedField isHidden={isHidden} />
```

Comment:
can we/should we useField this inside advanced field?

Reply:
Yeah I like that idea better. Updated